kai/rpc_server/server.py

```diff
@@ -463,7 +456,9 @@ class GetCodeplanAgentSolutionParams(BaseModel):
     file_path: Path
     incidents: list[ExtendedIncident]
 
+    # Deprecated in favor of llm_call_budget
```

Comment:
https://docs.pydantic.dev/latest/concepts/fields/#deprecated-fields You can make this a bonafide deprecated field if you want

Reply:
Or you could alias the other field to the new one

Reply:
+1 that's a good idea